### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by thewtex**
Improving top level docs.

It will be nice to have the package badges working after the first pre-release tag ;-).

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -5,11 +5,14 @@ scikit-build
 .. image:: https://badge.fury.io/py/scikit-build.png
```
Comment (jcfr) [inline README.rst]: In commit msg: AppVeyer -> AppVeyor

File: README.rst [hunk 2]
```diff
@@ -18,24 +18,16 @@ scikit-build
     :target: https://ci.appveyor.com/api/projects/status/github/scikit-build/scikit-build
 
 
-Simplify building Python extensions with CMake.  The scikit-build package is
-fundamentally just glue between distutils and CMake.  It wraps
-:code:`distutils.setup()` so that a CMake build step occurs before calling
-:code:`distutils.setup()`, and automatically appends CMake-built output into the
-call arguments as appropriate.
+Improved build system generator for Python C extensions. Better support is
```
Comment (jcfr) [inline README.rst]: ... for CPython ...

File: README.rst [hunk 3]
```diff
@@ -5,31 +5,29 @@ scikit-build
...
-  distutils setup.
+.. image:: https://ci.appveyor.com/api/projects/status/github/scikit-build/scikit-build
+    :target: https://ci.appveyor.com/api/projects/status/github/scikit-build/scikit-build
 
-* Passes command line options through to both CMake and distutils (tries to be
-  smart)
 
-* Uses information extracted from the CMake build to automatically provide most
-  of the arguments to :code:`setup()`.
+Improved build system generator for Python C extensions. Better support is
```
Comment (jcfr) [inline README.rst]: Two new line before `Better support` ...

**Comment by jcfr** [comment]
Looks good. Thanks.

(Few nitpicks that can definitively be done later)

File: README.rst (hunk 1 above)
Comment (thewtex) [inline README.rst]: Done

File: README.rst (hunk 2 above)
Comment (thewtex) [inline README.rst]: Done

File: README.rst (hunk 3 above)
Comment (thewtex) [inline README.rst]: Done

**Comment by jcfr** [comment]
Thanks

**Comment by jcfr** [comment]
Look like there are few issues with the integration of the badges: http://scikit-build.readthedocs.io/en/latest/readme.html

**Pull request merged**